examples/sushi/config.py

```diff
@@ -125,3 +125,19 @@
     default_test_connection=DuckDBConnectionConfig(catalogs=CATALOGS),
     model_defaults=ModelDefaultsConfig(dialect="duckdb"),
 )
+
+environment_catalog_mapping_config = Config(
+    default_connection=DuckDBConnectionConfig(
+        catalogs={
+            "physical": ":memory:",
```

Comment:
What do we need this one for?

Reply:
This is for the tests I run. The top catalog becomes the default catalog which I call "physical". I am testing to confirm that all the tables are created in "physical" and the views are created in their expected prod or dev catalogs.